CMakeLists.txt

```diff
@@ -695,24 +695,27 @@ if (BUILD_TESTING)
     if (S2N_INTEG_TESTS)
         find_package (Python3 COMPONENTS Interpreter Development)
         file(GLOB integv2_test_files "${PROJECT_SOURCE_DIR}/tests/integrationv2/test_*.py")
+        if (DEFINED ENV{CODEBUILD_BUILD_NUMBER})
```

Comment:
Can we use a CMake option instead of and env var?

We might need to add a dev shell to support this.